src/commands/backend/terminate.ts

```diff
@@ -4,19 +4,28 @@ import { lightMagenta } from '../../formatting'
 
 export default class Terminate extends Command {
   static description = 'Terminates a backend based on its backend name.'
+  // we turn off strict mode so that we can accept multiple values for the same argument
+  static strict = false
   static aliases = ['terminate']
   static examples = [
-    '<%= config.bin %> <%= command.id %> a8m32q',
+    '<%= config.bin %> <%= command.id %> abc123 def456 ...',
   ]
 
-  static args = [{ name: 'backend', required: true }]
+  static args = [{ name: 'backends', required: true }]
 
   public async run(): Promise<void> {
-    const { args } = await this.parse(Terminate)
-
+    // it's not possible to have a multiple values for the same argument, so this is the workaround
```

Comment:
a bit confused by this comment - could you help clarify?

Reply:
In the other commands we call parse() and then use the args returned from the command. Here, we can't do that because OCLIF doesn't support taking multiple vales for an arg. (In our case, we want to pass multiple backend values for the "backend" arg.) So instead, we are using this.argv directly.